src/common/common/network/net_command.h

```diff
@@ -68,7 +68,10 @@ enum class ClientCommand : command_t {
     // u8: The voxels mesh style aka VoxelMeshStyle
     // u8: The voxels state/type aka VoxelType
     // u8: Whether the block is collidable or not
-    GameRegistryData
+    GameRegistryData,
+
+    // Used for creation of command array
```

Comment:
Likely just me, but might wanna make this comment more explicit (i.e. "for getting the number of commands used").